go-controller/hybrid-overlay/pkg/controller/node_windows.go

```diff
@@ -31,13 +33,15 @@ type NodeController struct {
 	localNodeCIDR   *net.IPNet
 	localNodeIP     net.IP
 	remoteSubnetMap map[string]string // Maps a remote node to its remote subnet
+	nbClient        libovsdbclient.Client
 }
 
 // newNodeController returns a node handler that listens for node events
 // so that Add/Update/Delete events are appropriately handled.
 func newNodeController(kube kube.Interface,
 	nodeName string,
 	nodeLister listers.NodeLister,
+	nbClient libovsdbclient.Client,
```

Comment:
does libovsdb work with windows? :)

Reply:
Great question any ideas @dave-tucker?

Reply:
It compiles for Windows, and it should probably work, but I've not tested it.

Reply:
Once this is settled I can merge ds (before we merge this) and see how CI looks for HO

Reply:
As long as it compiles, we're fine. the Windows bits shouldn't use the client at all anyway.

Reply:
No libovsdb clients will be passed to any of the per node process code now